src/Stratis.Bitcoin.Features.Wallet/Wallet.cs

```diff
@@ -27,16 +27,22 @@ public Wallet()
         [JsonProperty(PropertyName = "name")]
         public string Name { get; set; }
 
+        /// <summary>
+        /// Flag indicating if it is a watch only wallet.
```

Comment:
But we already have a feature which is a watch only wallet. Should we remove this feature if it's logic is now here?

Reply:
No (discussed privately)